test/extended/util/annotate/rules.go

```diff
@@ -146,6 +146,12 @@ var (
 			`\[sig-storage\] In-tree Volumes \[Driver: azure\] \[Testpattern: Inline-volume`,
 			`\[sig-storage\] In-tree Volumes \[Driver: azure\] \[Testpattern: Pre-provisioned PV`,
 		},
+		"[Skipped:ovirt]": {
+			// https://bugzilla.redhat.com/show_bug.cgi?id=1838751
+			`\[sig-network\] Networking Granular Checks: Services should function for endpoint-Service`,
+			`\[sig-network\] Networking Granular Checks: Services should function for node-Service`,
+			`\[sig-network\] Networking Granular Checks: Services should function for pod-Service`,
+		},
```

Comment:
i don't believe this does what you want.  you need an actual change in the test to invoke skip when the ovirt platform is detected.

see that https://github.com/openshift/origin/pull/25012/files#diff-6ba77494282f6e840f44b01ce97335afR300 will not match this.

Reply:
I think that is the wrong URL

Reply:
it's right but github doesn't autoexpand for you.

line 300 of this file.

see also:
https://github.com/openshift/origin/blob/master/test/extended/oauth/requestheaders.go#L66-L68

as an example of how this is normally done.

Reply:
Looking at other PRs such as:
- https://github.com/openshift/origin/pull/24833/files
- https://github.com/openshift/origin/pull/24797/files

Are you sure I don't need to just edit zz_generated.annotations.go ?

Reply:
I am reasonably certain what you've done here will accomplish nothing other than adding a `[Skipped:ovirt]` prefix to the tests in question.

Reply:
I think that its what needs to be done, I looked into:
https://github.com/openshift/origin/blob/master/cmd/openshift-tests/provider.go#L57-L63
and it seems that the tests will be skipped if we have [Skipped:ovirt] in them.

Plus the tests come from kubernetes so I don't think we need to send a patch there:
https://github.com/kubernetes/kubernetes/blob/2fd8debe9b913f18b20a2269117409212391ef19/test/e2e/network/networking.go#L162

Reply:
Thanks, https://github.com/openshift/origin/blob/master/cmd/openshift-tests/provider.go#L57-L63 is the piece i was missing

Reply:
hack/update-generated-bindata.sh should update the other file for you